src/Sylius/Behat/Context/Setup/OrderContext.php

```diff
@@ -762,6 +827,30 @@ private function checkoutUsing(
         $this->proceedSelectingShippingAndPaymentMethod($order, $shippingMethod, $paymentMethod);
     }
 
+    private function createShippingPaymentMethodsAndAddress(): void
+    {
+        /** @var AddressInterface $address */
+        $address = $this->addressFactory->createNew();
+        $address->setCity('Wawa');
+        $address->setCountryCode($this->countryRepository->findOneBy([])->getCode());
+        $address->setFirstName('Jon');
+        $address->setLastName('Doe');
+        $address->setPostcode('000');
+        $address->setStreet('Happy');
+
+        $this->theCustomerAddressedItToWithIdenticalBillingAddress($address);
+
+        $shippingMethod = $this->shippingMethodRepository->findOneBy([]);
+
+        Assert::notNull($shippingMethod, 'Shipping method not found');
```

Comment:
IMO unnecessary message

Reply:
I would leave it, still better than `Expected value other than null` :D